pages/common/git-branch.md

```diff
@@ -7,6 +7,10 @@
 
 `git branch`
 
+- Show the name of the current branch:
```

Comment:
I'd rather move this under the 2nd example (`git branch -a`), since 1st and 2nd examples are connected.

Reply:
Sure.